src/test/java/com/klarna/hiverunner/ShellFindFilesTest.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) 2013-2021 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import com.klarna.hiverunner.annotations.HiveSQL;
+
+@ExtendWith(HiveRunnerExtension.class)
+public class ShellFindFilesTest {
```

Comment:
```suggestion
public class HiveRunnerExtesionTest {
```
^ let's make this test a bit more generic so we can use it in future for testing the general behaviour of the extension. I already think this project has too many test classes and they could be grouped together better but that's a different issue.